### ISSUE TITLE
jeager collector always connect to kafka with address “127.0.0.1”, regardless of whether the KAFKA_PRODUCER_BROKERS or --kafka.producer.brokers parameter is set or not.

### ISSUE THREAD

**Issue by jiapeng123456** [jaegertracing/jaeger#5475]
### What happened?

jeager collector always connect to kafka with address “127.0.0.1”, regardless of whether the KAFKA_PRODUCER_BROKERS or --kafka.producer.brokers parameter is set or not.

jaeger collector version: 1.32/1.57

### Steps to reproduce

jaeger collector version: 1.32/1.57。

command:
```
docker run --rm --name collector \
  -e SPAN_STORAGE_TYPE=kafka \
  -e KAFKA_PRODUCER_BROKERS="10.17.33.243:9092" \
  -e KAFKA_TOPIC="jaeger-spans" \
  jaegertracing/jaeger-collector:1.32

or

docker run --rm --name collector \
  -e SPAN_STORAGE_TYPE=kafka \
  -e KAFKA_TOPIC="jaeger-spans" \
  jaegertracing/jaeger-collector:1.32 \
  --kafka.producer.brokers="10.17.33.243:9092"
```

error:
```
{"level":"error","ts":1716368159.3924165,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
```

### Expected behavior

jeager collector connect to kafka with address specified by KAFKA_PRODUCER_BROKERS or --kafka.producer.brokers

### Operating system

macos

### Deployment model

docker

**Comment by yurishkuro** [comment]
```
$ docker run --rm --name collector \
  -e SPAN_STORAGE_TYPE=kafka \
  -e KAFKA_TOPIC="jaeger-spans" \
  jaegertracing/jaeger-collector:1.57 \
  --kafka.producer.brokers="127.0.0.100:9092"
```

fails with: {"level":"fatal","ts":1716393755.6846683,"caller":"collector/main.go:83","msg":"Failed to init storage factory","error":"kafka: client has run out of available brokers to talk to: dial tcp **127.0.0.100:9092**: connect: connection refused" ...

So works as expected in the latest version. We do not support 2yr old versions.

**Comment by jiapeng123456** [comment]
@yurishkuro   thanks for reply.

But the latest version (1.57) still outputs the same error for me.
The operations are as follows:

1, I changed the IP address based on the command above.
```
docker run --rm --name collector \
  -e SPAN_STORAGE_TYPE=kafka \
  -e KAFKA_TOPIC="jaeger-spans" \
  jaegertracing/jaeger-collector:1.57 \
  --kafka.producer.brokers="10.17.33.243:9092"
```

2, It started up normally. Logs are as follows:

```
2024/05/23 02:40:53 maxprocs: Leaving GOMAXPROCS=5: CPU quota undefined
2024/05/23 02:40:53 application version: git-commit=55e991a29725468164b11be5fc4e260dc09598d6, git-version=v1.57.0, build-date=2024-05-01T23:19:12Z
{"level":"info","ts":1716432053.6064036,"caller":"flags/service.go:110","msg":"Mounting metrics handler on admin server","route":"/metrics"}
{"level":"info","ts":1716432053.606551,"caller":"flags/service.go:116","msg":"Mounting expvar handler on admin server","route":"/debug/vars"}
{"level":"info","ts":1716432053.6068342,"caller":"flags/admin.go:130","msg":"Mounting health check on admin server","route":"/"}
{"level":"info","ts":1716432053.6068966,"caller":"flags/admin.go:144","msg":"Starting admin HTTP server","http-addr":":14269"}
{"level":"info","ts":1716432053.606934,"caller":"flags/admin.go:122","msg":"Admin server started","http.host-port":"[::]:14269","health-status":"unavailable"}
{"level":"info","ts":1716432053.6079986,"caller":"kafka/factory.go:77","msg":"Kafka factory","producer builder":{"Brokers":["10.17.33.243:9092"],"RequiredAcks":1,"Compression":"none","CompressionLevel":0,"ProtocolVersion":"","BatchLinger":0,"BatchSize":0,"BatchMinMessages":0,"BatchMaxMessages":0,"MaxMessageBytes":1000000,"Authentication":"none","Kerberos":{"ServiceName":"kafka","Realm":"","UseKeyTab":false,"Username":"","ConfigPath":"/etc/krb5.conf","KeyTabPath":"/etc/security/kafka.keytab","DisablePAFXFast":false},"TLS":{"Enabled":false,"CAPath":"","CertPath":"","KeyPath":"","ServerName":"","ClientCAPath":"","CipherSuites":null,"MinVersion":"","MaxVersion":"","SkipHostVerify":false,"ReloadInterval":0},"PlainText":{"Username":"","Mechanism":"PLAIN"}},"topic":"jaeger-spans"}
{"level":"info","ts":1716432053.6155412,"caller":"static/strategy_store.go:68","msg":"No sampling strategies source provided, using defaults"}
{"level":"info","ts":1716432053.6208608,"caller":"grpc@v1.63.2/server.go:675","msg":"[core][Server #1]Server created","system":"grpc","grpc_log":true}
{"level":"info","ts":1716432053.6214528,"caller":"server/grpc.go:104","msg":"Starting jaeger-collector gRPC server","grpc.host-port":"[::]:14250"}
{"level":"info","ts":1716432053.621793,"caller":"server/http.go:56","msg":"Starting jaeger-collector HTTP server","http host-port":":14268"}
{"level":"info","ts":1716432053.622567,"caller":"grpc@v1.63.2/server.go:871","msg":"[core][Server #1 ListenSocket #2]ListenSocket created","system":"grpc","grpc_log":true}
{"level":"info","ts":1716432053.6246357,"caller":"app/collector.go:146","msg":"Not listening for Zipkin HTTP traffic, port not configured"}
{"level":"info","ts":1716432053.6247103,"caller":"handler/otlp_receiver.go:77","msg":"OTLP receiver status change","status":"StatusStarting"}
{"level":"warn","ts":1716432053.6247692,"caller":"internal@v0.98.0/warning.go:42","msg":"Using the 0.0.0.0 address exposes this server to every network interface, which may facilitate Denial of Service attacks. Enable the feature gate to change the default and remove this warning.","documentation":"https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/security-best-practices.md#safeguards-against-denial-of-service-attacks","feature gate ID":"component.UseLocalHostAsDefaultHost"}
{"level":"info","ts":1716432053.6248076,"caller":"grpc@v1.63.2/server.go:675","msg":"[core][Server #3]Server created","system":"grpc","grpc_log":true}
{"level":"info","ts":1716432053.625252,"caller":"otlpreceiver@v0.98.0/otlp.go:102","msg":"Starting GRPC server","endpoint":"0.0.0.0:4317"}
{"level":"warn","ts":1716432053.6253357,"caller":"internal@v0.98.0/warning.go:42","msg":"Using the 0.0.0.0 address exposes this server to every network interface, which may facilitate Denial of Service attacks. Enable the feature gate to change the default and remove this warning.","documentation":"https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/security-best-practices.md#safeguards-against-denial-of-service-attacks","feature gate ID":"component.UseLocalHostAsDefaultHost"}
{"level":"info","ts":1716432053.625767,"caller":"otlpreceiver@v0.98.0/otlp.go:152","msg":"Starting HTTP server","endpoint":"0.0.0.0:4318"}
{"level":"info","ts":1716432053.6258106,"caller":"healthcheck/handler.go:129","msg":"Health Check state change","status":"ready"}
{"level":"info","ts":1716432053.6259253,"caller":"grpc@v1.63.2/server.go:871","msg":"[core][Server #3 ListenSocket #4]ListenSocket created","system":"grpc","grpc_log":true}
```

3, I made HotRod send some spans to the collector via the agent. However, the collector did not send spans to Kafka. It looked like the collector still sent spans to "127.0.0.1" instead of the specified IP address.

```
{"level":"error","ts":1716432066.992261,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
{"level":"error","ts":1716432066.9923155,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
{"level":"error","ts":1716432066.9923236,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
{"level":"error","ts":1716432066.992327,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
{"level":"error","ts":1716432066.99233,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
{"level":"error","ts":1716432066.9923327,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
{"level":"error","ts":1716432066.9923358,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
{"level":"error","ts":1716432066.9923398,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
{"level":"error","ts":1716432066.9923427,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
{"level":"error","ts":1716432066.9923456,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
{"level":"error","ts":1716432066.9923494,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
{"level":"error","ts":1716432066.9923685,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
{"level":"error","ts":1716432066.992374,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
{"level":"error","ts":1716432066.9923766,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
{"level":"error","ts":1716432066.9923832,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
{"level":"error","ts":1716432067.0973601,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
{"level":"error","ts":1716432067.097439,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
{"level":"error","ts":1716432067.0974586,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
{"level":"error","ts":1716432067.0974643,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
{"level":"error","ts":1716432067.0974686,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
{"level":"error","ts":1716432067.0974734,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
{"level":"error","ts":1716432067.0974853,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
{"level":"error","ts":1716432067.097489,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
{"level":"error","ts":1716432067.097493,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
{"level":"error","ts":1716432067.0974967,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
{"level":"error","ts":1716432067.0975006,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
{"level":"error","ts":1716432067.097504,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
{"level":"error","ts":1716432067.097508,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
{"level":"error","ts":1716432067.0975115,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
{"level":"error","ts":1716432067.990434,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
{"level":"error","ts":1716432067.9906595,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
{"level":"error","ts":1716432067.9907093,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
{"level":"error","ts":1716432067.9907155,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
{"level":"error","ts":1716432067.9907193,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
{"level":"error","ts":1716432067.990723,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
{"level":"error","ts":1716432067.990727,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
{"level":"error","ts":1716432067.9907298,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
{"level":"error","ts":1716432067.9907334,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
{"level":"error","ts":1716432067.9907365,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
{"level":"error","ts":1716432067.99074,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
{"level":"error","ts":1716432067.990743,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
{"level":"error","ts":1716432067.9907475,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
{"level":"error","ts":1716432067.990752,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
{"level":"error","ts":1716432067.9907558,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
{"level":"error","ts":1716432067.9907594,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
{"level":"error","ts":1716432067.9907641,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
{"level":"error","ts":1716432067.9907677,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
{"level":"error","ts":1716432068.0949357,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
{"level":"error","ts":1716432068.094987,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
{"level":"error","ts":1716432068.0949926,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}
{"level":"error","ts":1716432068.0949955,"caller":"kafka/writer.go:61","msg":"dial tcp 127.0.0.1:9092: connect: connection refused","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\tgithub.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:61"}

```
